### ISSUE TITLE
Display folder name when using search

### ISSUE THREAD

**Issue by comatory** [mssun/passforios#499]
Currently when I use search to filter results, I get all the items correctly. However sometimes I have passwords with same names under different folders. It's impossible to differentiate between them when search is used. 
When search is not used and "show folders" is turned on, folder names are displayed in list view. I think same style of display could be used when filtering.

**Comment by SimplyDanny** [comment]
The search focuses on password names, not on folders. You can see the folder in which the password resides on the right hand side in the search list. Showing folders in the search list would slow down the selection of a password, because you would have to navigate through the hierarchy of folders first.

**Comment by comatory** [comment]
I'm not really asking to search for folders, only to display them. Right now I search for "google.com" and get 6 results in list view without any idea which one belongs to what, not the greatest UX.

I can see two ways of doing this:

1. Show folder name on the list item next to the password name. That can be problematic when password names are long so either folder name or password name would have to be shortened.
2. Group the search results by folder name if password is found within multiple folders. 

@SimplyDanny Yeah I wouldn't want that, I would assume that whatever the filtered list is, each list item belongs to password and not a folder.

Of course I'm not sure if this is feasible technically as this is only a suggestion (unfortunately I can't write in Swift/ObjC). Solution no. 1 would be just fine for me.

**Comment by SimplyDanny** [comment]
No. 1 is already implemented. On the right-hand side you see the folder path:
![Screenshot 2021-09-02 at 16 37 53](https://user-images.githubusercontent.com/16365760/131863809-251a0afd-19b3-4a83-9ee7-73e9b14ee1a2.png)
There are two password named `github.com`. One resides in `folder`, the other one in `folder/it`.

**Comment by comatory** [comment]
That is strange because I'm not seeing this. 
Could it be because of my pass directory structure?

I have google.com in my pass directory and then I have `a/google.com` (meaning `a` subfolder) inside.

**Comment by SimplyDanny** [comment]
Strange. It doesn't actually matter where you put the files. You always get the folder path on the right.

The feature is relatively old, but just to be sure: Which version are you using and on which device category (iPhone, iPad, ...)?

**Comment by comatory** [comment]
0.11.0 (11) on iPhone 8 Plus and iOS 14.7.1

**Comment by SimplyDanny** [comment]
You are right. The folder paths are missing in this version. As can be seen in the screenshot above, this is definitely fixed in the current beta version (0.12.0 (33)).

**Comment by jhthorsen** [comment]
When will the next version be available?
